src/services/data-service.js

```diff
@@ -0,0 +1,46 @@
+import { Post } from '../models/post';
+
+import firebaseReference from "../firebaseConfig"
+import 'firebase/firestore';
+
+export class DataService {
+	constructor(numPosts) {
+		this.numPosts = numPosts
+    }
+    
+
+	getPostsFromDB() {
+
+        var posts = [];
+
+        var data = firebaseReference.collection('days').get()
```

Comment:
idk how firestore works, but if 'days' is the collection, should it get the days instead of the posts? Correct me if I''m wrong.

Reply:
ahh okay that's because I used "days" as the name for my test database. It's fixed because I just renamed it to posts.